source/Calamari.Common/Features/Scripting/DotnetScript/Bootstrap.csx

```diff
@@ -102,14 +102,12 @@ public static class Octopus
 
         public string DecryptString(string encrypted, string iv)
         {
-            using (var algorithm = new AesCryptoServiceProvider()
-            {
-                Mode = CipherMode.CBC,
-                Padding = PaddingMode.PKCS7,
-                KeySize = 128,
-                BlockSize = 128
-            })
+            using (var algorithm = Aes.Create())
```

Comment:
https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.aescryptoserviceprovider?view=net-6.0
`Derived cryptographic types are obsolete. Use the Create method on the base type instead.`